src/routing/signOutRouteGuard.ts

```diff
@@ -34,9 +34,13 @@ export class SignOutRouteGuard implements RouteGuard {
                         const identity = await this.mapiClient.get<Identity>("/identity", [await this.mapiClient.getPortalHeader("delegationSignOut")]);
 
                         if (identity) {
-                            const redirectUrl = await this.backendService.getDelegationUrl(DelegationAction.signOut, { userId: identity.id });
+                            const delegationParam = {};
```

Comment:
In the other comment I'm suggesting to hide the composition..
```
const delegationParam = {};
delegationParam[DelegationParameters.UserId] =  identity.id;
const redirectUrl = await this.backendService.applyDelegation(DelegationAction.signOut, delegationParam);
```
inside specifically named methods, like `getSignoutDelegationUrl` or something like that.